### ISSUE TITLE
Black line appears under pinned tabs on Linux

### ISSUE THREAD

**Issue by Slater91** [black7375/Firefox-UI-Fix#117]
**Describe the bug**
A black line appears under pinned tabs on Linux.

**Expected behavior**
No line appears, just like with non-pinned tabs.

**Screenshots**
![Screenshot_20210705_143819](https://user-images.githubusercontent.com/11941225/124472639-448f3180-dd96-11eb-9d80-0d5d3cdab25f.png)

**Environment (please complete the following information):**
Check like `- [x]`.
 - Distribution
   - [x] [Original Lepton](https://github.com/black7375/Firefox-UI-Fix)
   - [ ] [Lepton's photon style](https://github.com/black7375/Firefox-UI-Fix/tree/photon-style)
 - Firefox Version: [write from `about:support` - `version`]
 - OS:
   - [x] Linux
   - [ ] Windows
   - [ ] Mac
   - [ ] Others: 
 - Theme:
   - [ ] Light
   - [ ] Dark
   - [ ] Alpenglow(System: Light)
   - [ ] Alpenglow(System: Dark)
   - [x] System Default
     - GTK Theme Name(Only Linux User): Qogir

**Comment by black7375** [comment]
Are you using `touch mode`?
I checked it in touch mode.

**Comment by Slater91** [comment]
No, I am using normal mode. The line is also present in compact mode, while in touch mode pinned tabs are simply detached from the tab bar.

**Comment by black7375** [comment]
**Reproduce**
I couldn't reproduce it in KDE, so I installed him and succeeded.
It only happens when overflow occurs.
![Gnome none overflow](https://user-images.githubusercontent.com/25581533/124690751-2b879d00-deca-11eb-92d5-c39161df703c.png)
![Gnome overflow](https://user-images.githubusercontent.com/25581533/124690760-2de9f700-deca-11eb-99ad-27c41c238cfc.png)

**Result**
It has been patched considering compatibility with other themes and seems to have been fixed now.  https://github.com/black7375/Firefox-UI-Fix/commit/322b41c630b9958b7a3b3a04b31dada544748227
![Gnome patched none overflow](https://user-images.githubusercontent.com/25581533/124690767-2fb3ba80-deca-11eb-9da8-c20ba9e64fad.png)
![Gnome patched overflow](https://user-images.githubusercontent.com/25581533/124690772-317d7e00-deca-11eb-86f4-9cbed80d2e57.png)

**More**
if it still hasn't been fixed..
Here's the code causing it:
```css
#nav-bar:not([tabs-hidden="true"]) {
  box-shadow: 0 calc(-1 * var(--tabs-navbar-shadow-size)) 0 var(--tabs-border-color);
}
```

This will force the line to be removed.
```css
#nav-bar:not([tabs-hidden="true"]) {
  --tabs-border-color: var(--toolbar-bgcolor);
}
```

---

Please let me know if it has been fixed. Thanks for report.

**Comment by Slater91** [comment]
I can confirm version 2.2.1 fixes the issue. Thank you!
